packages/botonic-react/src/components/multichannel/multichannel-text.jsx

```diff
@@ -120,45 +132,104 @@ export const MultichannelText = props => {
     const buttonsTextSeparator =
       props.buttonsTextSeparator || DEFAULT_WHATSAPP_MAX_BUTTON_SEPARATOR
 
+    //MORE THAN 3 BUTTONS
     if (
       !postbackButtonsAsText &&
       postbackButtons.length > WHATSAPP_MAX_BUTTONS
     ) {
```

Comment:
Instead of adding a comment, why not creating an `exceedWhatsAppMaxButtonNumber` (or something like this) function that executes this condition?

Reply:
ok. I create a exceedWhatsAppMaxButtonNumber